main.go

```diff
@@ -68,7 +83,7 @@ func startGraphql(r *mux.Router) {
 	})
 
 	// serve the GraphQL endpoint at "/graphql"
-	r.Handle("/graphql", h)
+	r.Handle("/graphql", CorsMiddleware(h))
```

Comment:
Remove CorsMiddleware registration from graphql, it is done on the top level router.

Reply:
Fixed